frontend/src/metadata/cells/metadata-cells/ContinuousMetadataCell.svelte

```diff
@@ -67,6 +63,6 @@
 	<VegaLite
 		bind:view
 		spec={continuousVegaSpec($metricRange)}
-		data={chartData}
+		data={{ table: histogram.map((h) => Object.assign({}, h)) }}
```

Comment:
Is this a vega-lite hack so it removes the vega keys (so it actually reloads)? If not, what is this Object.assign jazz for?

Reply:
Exactly what it is, I had a nightmare trying to get this to work. Added a clarifying comment